custom_typings/merge-stream.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module "merge-stream" {
```

Comment:
To get into `@types` you want to send a PR to the `types-2.0` branch, not master